web/src/components/__tests__/RemoteContent.spec.tsx

```diff
@@ -50,4 +50,22 @@ describe('RemoteContent', () => {
 
     expect(onInternalLinkClick).toHaveBeenCalledTimes(0)
   })
+
+  it('should sanitize the html', () => {
```

Comment:
Did you test whether this test really fails if you remove the dom purify sanitization?

Reply:
Adjusted it.